variables.tf

```
@@ -523,3 +523,9 @@ variable "security_group_ids" {
   type        = list(string)
   default     = []
 }
+
+variable "use_old_arn_format" {
+  type        = bool
+  description = "A flag to enable/disable tagging the ecs resources that require the new longer arn format"
```

```suggestion
  description = "A flag to enable/disable tagging the ecs resources that require the new longer arn format"
  type        = bool
```

Minor suggestion, just to follow the code styling :smile: